client/src/utils/date.js

```diff
@@ -1,3 +1,6 @@
 export function getCurrentDateString() {
   return new Date().toISOString().substr(0, 10);
 }
+export function getCurrentDateShort() {
+  return new Date().toLocaleDateString().substr(0, 7);
```

Comment:
`.toLocaleDateString` is problematic here because, these get formatted differently for different standard languages on mobile phones for example. Keep that in mind.